### DISCUSSION TITLE
UniGetUI 3.1.2 beta 1

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#2707] [Announcements]
![](https://img.shields.io/github/downloads/marticliment/wingetui/3.1.2-beta1/total?style=for-the-badge&color=red)

Note: This release is not digitally signed for convenience. Stable releases will always be signed.

# Changes
 - The "Package Details" page will offer more actions when a package is opened.
 - The filters is now fully closable, with a new toggle button located on the left side of the search bar
 - The LanguageEngine is more resilient to string deletion on the updated language files.
 - Automated package reload in the installed packages page is now less agressive. It will not remove the packages, only add any missing ones
 - Improvements to the Package Bundles page:
   - UniGetUI now supports the `.ubundle` extension, which will ease the process of opening package bundles. The file is internally JSON.
   - UniGetUI will complain if a bundle fails to be created.
   - UniGetUI will complain if an unsaved bundle is going to be lost. (The user wants to create a new one, etc)
   - UniGetUI will warn you if you are overriding a file
   - Improved handling of InstallationOptions
   - Bundles will show the version set to install, or the text "Latest". It used to show the version with which a package was exported.
 - Optimize how packages are loaded when shared from the "Share package" website.
 - UniGetUI can now be interacted with using the `unigetui://` protocol. More details here: https://github.com/marticliment/UniGetUI/blob/main/cli-arguments.md
 - Other improvements and bug fixes

SHA256: `27B4E938560BACC58CBD5DE4878EE7033AABA023F53226BAC4FDE0315E7DE793`

<hr /><em>This discussion was created from the release <a href='https://github.com/marticliment/UniGetUI/releases/tag/3.1.2-beta1'>UniGetUI 3.1.2 beta 1</a>.</em>

**Comment by phoenixlucky** [comment]
Why only the source code? there is no installation package.

**Reply by ctan13** [reply]
was wondering the same thing

**Reply by RonnyTNL** [reply]
I'm guessing the build script failed @marticliment ?

**Reply by marticliment** [reply]
I just forgot to upload them, they should be up now

**Reply by RonnyTNL** [reply]
Yes, thanks.
Running a scan works fine, application exits clean 👍

**Comment by shanto** [comment]
Thanks for the pre-release fixing #2615 among other good ones.
